src/sql/workbench/contrib/extensions/electron-browser/localizedConstants.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from 'vs/nls';
+
+// localizable strings
+
+export const visualizerExtensions = 'visualizerExtensions';
```

Comment:
This isn't being localized - and given the context it doesn't seem like it should be so I'd suggest just having this file be named 'constants.ts'